### PR TITLE
GH Actions, Issue Template, and README.md Overhaul

### PR DESCRIPTION
**Pull request by BeansBeefBroccoli**
🎺 Doot

### CODE REVIEW

**Comment by CammiePone** [comment]
Generally it's best to make separate PRs for different features

File: README.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
+| ---------------------------- | ------------------------------------------------------- | ----------------------------------------------------------------------- | ------------------------------------------- |
+| State                        | Released                                                | Released                                                                | Pre-Alpha                                   |
+| Platform                     | Bedrock                                                 | Java                                                                    | Java                                        |
+| Mod Support                  | Bedrock Add-Ons                                         | Forge                                                                   | Fabric                                      |
+| Performance                  | :1st_place_medal:                                       | :2nd_place_medal:                                                       | :3rd_place_medal: (With Sodium)             |
```
Comment (leon0399) [inline README.md:26]: I think you misplaced medals for Vivecraft and MCXR

Comment (BeansBeefBroccoli) [reply README.md]: Fixed!

**Comment by BeansBeefBroccoli** [comment]
@CammiePone 

> Generally it's best to make separate PRs for different features

Yes, I was going to make two branches, but I was being dumb and didn't notice what branch I was contributing to.

**Comment by BeansBeefBroccoli** [comment]
Also, I'm marking this as a draft cause it seems to be broken.

**Comment by Fishvap** [comment]
Not a fan of this readme imo. The rickroll link on the logo has already put me off.

**Comment by BeansBeefBroccoli** [comment]
ok lol I'll take it off but what else do you not like about it, I'm open to feedback

**Comment by Fishvap** [comment]
Personally I just don't feel it's necessary to update the Readme when the mod hasn't been released yet. 😊

**Comment by BeansBeefBroccoli** [comment]
Ok, I understand

**Comment by BeansBeefBroccoli** [comment]
Ok so, the actions are currently failing, but I don't think that's it's problem, so, I'm un-marking this as a draft.

**Comment by BeansBeefBroccoli** [comment]
Oops

**Comment by BeansBeefBroccoli** [comment]
Closed cause it's outdated

**Pull request closed without merging**